src/hooks/useExistInFeature.js

```diff
@@ -0,0 +1,40 @@
+import {useEffect, useState} from 'react'
+import {useSearchParams} from 'react-router-dom'
+import debug from '../utils/debug'
+
+
+/**
+ * Feature Hook
```

Comment:
`This hook checks for the existence of a named feature in the URL SearchParams (via react-router), e.g. `feature=app,placemark`.`

Reply:
done